### PR TITLE
add debian and ubuntu packages

### PR DESCRIPTION
**Pull request by alexmyczko**
can be closed now? https://github.com/jstaf/onedriver/issues/396

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,18 @@ other OneDrive clients:
 
 ## Quick start
 
+### Debian/Ubuntu
```
Comment (foopsss) [inline README.md:82]: The header is probably good enough as it was previously.

File: README.md [hunk 2]
```diff
@@ -79,6 +79,18 @@ other OneDrive clients:
...
+
+Users of Debian (soon available via backports for Debian 12) and Ubuntu 24.04 or later
+are advised to use the official package. 
+
+Debian: https://packages.debian.org/onedriver
```
Comment (foopsss) [inline README.md:87]: I'm not sure these links are really needed, since most users won't probably have much of a use for them. I'd take them out.

**Comment by foopsss** [comment]
I'd also rename the PR to something like "Mention distro packages for Debian/Ubuntu" for the sake of clarity.

**Comment by alexmyczko** [comment]
feel free to edit as you wish

**Pull request closed without merging**